### PR TITLE
Added minor changes for bootcamp interoperability

### PR DESCRIPTION
**Pull request by azidar**
Provides minor changes for chisel-bootcamp, and adds publishing settings for build.sbt

### CODE REVIEW

**Comment by edwardcwang** [comment]
Did it already get published? Also, does this mean that diagrammer will no longer work with Chisel master?

**Comment by azidar** [comment]
No it hasn't been published, this is the PR intended to make it publishable. I'd like it to work in the bootcamp, which is not on chisel master, so I'm not sure what the right solution is. I think as is, it should still work with chisel-master.

File: build.sbt [hunk 1]
```diff
@@ -30,14 +37,46 @@ def javacOptionsVersion(scalaVersion: String): Seq[String] = {
...
+  }
+  else {
+    Some("releases" at nexus + "service/local/staging/deploy/maven2")
+  }
+}
 
 // Provide a managed dependency on X if -DXVersion="" is supplied on the command line.
 val defaultVersions = Map(
-  "chisel3" -> "3.2-SNAPSHOT"
+  "chisel3" -> "3.1.0"
```
Comment (edwardcwang) [inline build.sbt]:
```suggestion
  "chisel3" -> "3.1.+"
```

File: build.sbt [hunk 2]
```diff
@@ -30,14 +37,46 @@ def javacOptionsVersion(scalaVersion: String): Seq[String] = {
...
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+  <scm>
+    <url>https://github.com/freechipsproject/diagrammer.git</url>
+    <connection>scm:git:github.com/freechipsproject/diagrammer.git</connection>
+  </scm>
+  <developers>
+    <developer>
+      <id>jackbackrack</id>
```
Comment (ucbjrl) [inline build.sbt]: I think we should change this to chick/monica

File: build.sbt [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
-
-scalaVersion := "2.12.6"
+resolvers ++= Seq(
+  Resolver.sonatypeRepo("snapshots"),
+  Resolver.sonatypeRepo("releases")
+)
+organization := "edu.berkeley.cs"
+version := "1.0-SNAPSHOT"
+autoAPIMappings := true
+scalaVersion := "2.11.6"
```
Comment (ucbjrl) [inline build.sbt]: Could we make the "default" `scalaVersion` 2.12.7 (i.e., same as FIRRTL), and the `crossScalaVersions` ` Seq("2.12.7", "2.11.12")`?

**Comment by azidar** [comment]
Addressed comments @ucbjrl

**Pull request merged**